src/l10n/nl.js

```diff
@@ -0,0 +1,353 @@
+            'Import successful'                   : 'Import succesvol',
+            'Import error'                        : 'Import fout',
+            'Invalid file type "{type}"'          : 'Ongeldig bestandstype "{type}"',
+            'Password required'                   : 'Wachtwoord benodigd',
+            'Password invalid'                    : 'Ongeldig wachtwoord',
+            'Failed to decrypt passwords'         : 'Ontsleutelen van wachtwoorden mislukt',
+            'Failed to decrypt folders'           : 'Ontsleutelen van mappen mislukt',
+            'Failed to decrypt tags'              : 'Ontleutelen van Tags mislukt',
+            'CSV file can not be mapped'          : 'CSV bestand kan niet ingedeeld worden',
+            'Delete All Items'                    : 'Alle items verwijderen',
+            'Restore All Items'                   : 'Alle items restoren',
+            'Restore Items'                       : 'Restore Items',
+            'Restore all items in trash?'         : 'Alle items restoren in de prullenbak?',
+            'Restore this item'                   : 'Dit item restoren',
+            'Items restored'                      : 'Item restored',
+            'Empty Trash'                         : 'Prullenbak legen',
+            'Delete all items in trash?'          : 'Wilt u alle items in de prullenbak definitief verwijderen?',
+            'Trash emptied'                       : 'Prullenbak geleegd',
+            'Search'                              : 'Zoeken',
+            'Use the search box to search'        : 'Gebruik het zoekvenster om te zoeken',
+            'Search everywhere for "{query}"'     : 'Zoek overal naar "{query}"',
+            'Settings'                            : 'Instellingen',
+            'Password Rules'                      : 'Wachtwoord regels',
+            'Mark duplicates'                     : 'Markeer duplicaten',
+            'Maximum age in days'                 : 'Maximale leeftijd in dagen',
+            'Password Generator'                  : 'Wachtwoord generator',
+            'Password strength'                   : 'Wachtwoord sterkte',
+            'Include numbers'                     : 'Cijfers gebruiken',
+            'Include special characters'          : 'Bijzondere tekens gebruiken',
+            'User Interface'                      : 'Gebruikersinterface',
+            'Initial section'                     : 'Startpagina',
+            'All Passwords'                       : 'Alle wachtwoorden',
+            'Passwords List View'                 : 'Wachtwoorden in lijstweergave',
+            'Set title from'                      : 'Dit veld als titel gebruiken',
+            'Sort by'                             : 'Sorteer op',
+            'Title field'                         : 'Titelveld',
+            'Single click action'                 : 'Atie bij klik',
+            'Double click action'                 : 'Aktie bij dubbelklik',

```

Comment:
In line 249 its "Atie", please confirm

Reply:
Indeed, changed al the instances of " aktie" to "actie" (also the misspelled one "Atie". It seems that writing it with a " c"  is the preferred way.